sormas-rest/src/main/resources/openApiConfig.yml

```
@@ -0,0 +1,5 @@
+openAPI:
+  info:
+    title: SORMAS REST API
```

Let's add the following based on https://github.com/swagger-api/swagger-core/wiki/Swagger-2.X---Integration-and-Configuration#configuration-file

```suggestion
    title: SORMAS REST API
    description: https://github.com/hzi-braunschweig/SORMAS-Project/tree/development/sormas-rest
    license:
      name: GPL v3
      url: https://www.gnu.org/licenses/gpl-3.0.html
```


Also we should include a version (required field), but I'm not sure what's the best approach here. Would be nice to take the current version from sormas-base pom.